yarn.lock

```diff
@@ -8091,6 +8452,32 @@ klona@^2.0.4:
   resolved "https://registry.yarnpkg.com/klona/-/klona-2.0.5.tgz#d166574d90076395d9963aa7a928fabb8d76afbc"
   integrity sha512-pJiBpiXMbt7dkzXe8Ghj/u4FfXOOa98fPW+bihOJ4SjnoijweJrNThJfd3ifXpXhREjpoF2mZVH1GfS9LV3kHQ==
 
+knip@^2.41.5:
+  version "2.41.5"
+  resolved "https://registry.yarnpkg.com/knip/-/knip-2.41.5.tgz#3e9b088f2be3cc4fa33a6abeb425d3d9ef026a29"
+  integrity sha512-w6C0/tQvvRAWSRo4Vl3CRqmJujx+vizFD4sGWuhdpmx89micpfMZITiGCc54c6S9phGHvZeMNtYGEOyNwbfcjg==
+  dependencies:
+    "@ericcornelissen/bash-parser" "^0.5.2"
+    "@npmcli/map-workspaces" "^3.0.4"
+    "@pkgjs/parseargs" "0.11.0"
+    "@pnpm/logger" "5.0.0"
+    "@pnpm/workspace.pkgs-graph" "2.0.10"
+    "@snyk/github-codeowners" "^1.1.0"
+    chalk "^5.2.0"
+    easy-table "^1.2.0"
+    fast-glob "^3.2.12"
+    globby "^13.1.3"
+    jiti "1.21.0"
+    js-yaml "^4.1.0"
+    micromatch "^4.0.5"
+    minimist "^1.2.8"
+    pretty-ms "^8.0.0"
+    strip-json-comments "^5.0.0"
+    summary "^2.1.0"
+    typescript "^5.0.2"
+    zod "3.22.4"
+    zod-validation-error "1.5.0"
```

Comment:
guh

Reply:
ya. maybe i'll pop in an `npx` command into package.json instead

could just depend on how excited people are to rm dead code.